src/app/GUI/Expressions/expressiondialog.cpp

```diff
@@ -24,6 +24,7 @@
 // Fork of enve - Copyright (C) 2016-2020 Maurycy Liebner
 
 #include "expressiondialog.h"
+#include "../../../core/Animators/qrealanimator.h"
```

Comment:
There should be no reason to add ../... etc in includes

Reply:
Umm, I'll have a second look at it, I remembered to be forced to include it as I needed something...

Reply:
yes, it was unnecessary, removed!

Reply:
I meant do not use ../ for paths to includes, just reference them as core/whatever :)

Anyway, there is no need to do much on this PR yet, I need to finish the "backend" first, then this PR will need to be adjusted.

Reply:
yes ,don't worry, I just reviewed your comments and accepted them.

I don´t plan to do any further change, it's ready for your adjustments 😉